vendor/engines/bulk_email/app/models/bulk_email/job.rb

```diff
@@ -0,0 +1,20 @@
+module BulkEmail
+
+  class Job < ActiveRecord::Base
+
+    self.table_name = "bulk_email_jobs"
+
+    validates_presence_of :subject
+    validates_with JobJsonFieldValidator
+
+    def search_criteria=(value)
```

Comment:
I don't think you're setting this yet, but could we use ActiveRecord's built-in serialization? `serialize :search_criteria, JSON`

http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Serialization/ClassMethods.html